### PR TITLE
Issue 22 sara

### PR DESCRIPTION
**Pull request by Sara-webaplication**
please review my code and let me know what should change

### CODE REVIEW

File: photo-points/Models/Capture.cs [hunk 1]
```diff
@@ -13,8 +13,8 @@ public class Capture
     public DateTime captureDate { get; set; }
     public long photoPointID { get; set; }
     public long userID { get; set; }
-
-    public ICollection<Data> ppData { get; set; }
+        public bool Approved { get; set; }
```
Comment (weberwilliamb) [inline photo-points/Models/Capture.cs:16]: I already added this in my pending merge. So we should be good here.

File: photo-points/Services/AdminReviewServices.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+using System.Threading.Tasks;
+using photo_points.Models;
+
+namespace photo_points.Services
+{
+    // You may need to install the Microsoft.AspNetCore.Http.Abstractions package into your project
+    public class AdminReviewServices : IAdminReviewServices
+    {
+
+        public bool approve(long captureID, byte photo, DateTime captureDate, bool Approved)
```
Comment (weberwilliamb) [inline photo-points/Services/AdminReviewServices.cs]: Are you passing a the true parameter from the Approved?

**Review by sruble — APPROVED** [review]
Looks good Sara

File: photo-points/Services/IAdminReviewServices.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+using System.Threading.Tasks;
+using photo_points.Models;
+
+namespace photo_points.Services
+{
+    // You may need to install the Microsoft.AspNetCore.Http.Abstractions package into your project
+    public interface IAdminReviewServices
+    {
+
+        bool approve(long captureID, byte photo, DateTime captureDate, bool Approved);
```
Comment (Bansenauer-Cascadia) [inline photo-points/Services/IAdminReviewServices.cs]: This method only needs the captureID to do its work

**Review by Bansenauer-Cascadia — CHANGES_REQUESTED** [review]

File: photo-points/Services/AdminReviewServices.cs (hunk 2 above)
Comment (Sara-webaplication) [reply photo-points/Services/AdminReviewServices.cs]: yes

File: photo-points/Services/IAdminReviewServices.cs (hunk 3 above)
Comment (Sara-webaplication) [reply photo-points/Services/IAdminReviewServices.cs]: changes is done

**Review by weberwilliamb — APPROVED** [review]
Looks good.

**Comment by Sara-webaplication** [comment]
Eric will pull my branch and merge to develop after he is done with his codes

**Comment by Bansenauer-Cascadia** [comment]
After you get the combined file from Eric, be sure to use a byte array for your photo data in "Capture.cs", then you can load the images into your fake repository using
byte[] imgdata = System.IO.File.ReadAllBytes(HttpContext.Current.Server.MapPath(path));

Then Sim can use the method below to convert the byte[] back into an image
public Image byteArrayToImage(byte[] byteArrayIn)
{
    MemoryStream ms = new MemoryStream(byteArrayIn);
    Image returnImage = Image.FromStream(ms);
    return returnImage;
}

**Pull request closed without merging**